### ISSUE TITLE
Address "ghost positions" for the long term

### ISSUE THREAD

**Issue by manulera** [pombase/allele_qc#63]
Now I am using a `?` symbol to mark those residues, but I should probably think of something better.

**Comment by manulera** [comment]
Hi @kimrutherford, I guess you have implemented something in your script that ignores the `?` positions in the script that reads fixes to modifications (see comment in https://github.com/pombase/allele_qc/issues/66#issuecomment-1673419593).

This can also happen in allele descriptions, for instance, you can try `SPBC25B2.07c` `S4A` in the `old_coords_fix` endpoint of the API. `S4` existed in an old genome, but it maps to nothing in the current sequence, so the response would be:

```json
[
  {
    "values": "?",
    "revision": "1381",
    "location": "complement(2609038..2610591)"
  }
]
```

I suspect this would be very rare, since once you go into such detailed modifications you may already know the aminoacid sequence, but if an intron is added and affects a partial deletion, this could happen.

TLDR: if `change_description_to` contains a `?`, don't let the change go through.
